UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb

```diff
@@ -0,0 +1,948 @@
+{
```

Comment:
<p>Recommend to change the title to say what we are going to do with GUVcat.</p><p> </p><p>Also need some information, either here or back up at the intro, about what GUVcat is and how we will use it.</p><p> </p><p>I appreciate the explanation of the naming conventon! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>